images/macos/macos-10.15-Readme.md

```diff
@@ -110,19 +115,19 @@
 - Xcode Command Line Tools 12.4.0.0.1.1610135815
 
 ### Linters
-- SwiftLint 0.46.5
+- SwiftLint 0.47.0
 - yamllint 1.26.3
 
 ### Browsers
 - Safari 15.4 (15613.1.17.1.11)
 - SafariDriver 15.4 (15613.1.17.1.11)
-- Google Chrome 99.0.4844.83 
+- Google Chrome 100.0.4896.60 
 - ChromeDriver 99.0.4844.51
```

Comment:
@mikhailkoliada @igorboskovic3 not sure if this is ok to have this mismatch between chrome and driver versions